.builds/freebsd.yml

```diff
@@ -0,0 +1,18 @@
+image: freebsd/latest
+sources:
+  - https://github.com/sunfishcode/cap-std
+tasks:
+  - setup: |
+      curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y --default-toolchain=none --profile=minimal
+      source $HOME/.cargo/env
```

Comment:
Should `$HOME` be whitespace-protected by doublequotes? Or can we use `~` instead? (here and below)